app/src/main/java/org/mozilla/fenix/components/TabCollectionStorage.kt

```diff
@@ -59,6 +60,7 @@ class TabCollectionStorage(
     fun createCollection(title: String, sessions: List<Session>) {
         collectionStorage.createCollection(title, sessions)
         notifyObservers { onCollectionCreated(title, sessions) }
+        Collections.totalCollections.set(getTabCollectionsCount().toLong())
```

Comment:
Did we want a histogram with number of total collections and pages per collection?

Reply:
I think we are omitting the histogram for now? @ecsmyth might be able to confirm this?